src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/PrisonDto.kt

```diff
@@ -10,7 +10,7 @@ data class PrisonDto(
   var code: String,
 
   @Schema(description = "is prison active", example = "true", required = true)
-  var active: Boolean,
+  var active: Boolean = false,
 
   @Schema(description = "exclude dates", required = false)
   var excludeDates: List<LocalDate> = mutableListOf(),
```

Comment:
sorry not part of this change but this could be a Set to avoid duplicates.

Reply:
good point I will change